airbyte-integrations/connectors/destination-iceberg-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/iceberg/v2/IcebergV2WriteTest.kt

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.iceberg.v2
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.airbyte.cdk.load.test.util.NoopDestinationCleaner
+import io.airbyte.cdk.load.test.util.NoopExpectedRecordMapper
+import io.airbyte.cdk.load.write.BasicFunctionalityIntegrationTest
+import io.airbyte.cdk.load.write.StronglyTyped
+import java.util.Base64
+import okhttp3.*
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+
+abstract class IcebergV2WriteTest(configContents: String) :
+    BasicFunctionalityIntegrationTest(
+        configContents,
+        IcebergV2Specification::class.java,
+        IcebergV2DataDumper,
+        NoopDestinationCleaner,
+        NoopExpectedRecordMapper,
+        // TODO let's validate these - I'm making some assumptions about how iceberg works
```

Comment:
```suggestion
```
I guess we've validated them sufficiently? :shrug: if tests fail later on, we can always modify here